### ISSUE TITLE
Cannot create a git push

### ISSUE THREAD

**Issue by Mikel-Landa** [microsoft/azure-devops-rust-api#352]
From the spec: https://learn.microsoft.com/en-us/rest/api/azure/devops/git/pushes/create?view=azure-devops-rest-7.1&tabs=HTTP#gitchange

![image](https://github.com/microsoft/azure-devops-rust-api/assets/51875803/8eb86dc6-1f2d-4a55-8bf3-e0357d641a03)

However, the change type in rust:
```rs
#[doc = ""]
#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
pub struct Change {
    #[doc = "The type of change that was made to the item."]
    #[serde(rename = "changeType")]
    pub change_type: change::ChangeType,
    pub item: serde_json::Value,
}
```

Most of the fields are missing, so a push cannot be created.

**Comment by johnbatty** [comment]
Thanks very much for reporting the issues.  I'll make a fix...

**Comment by johnbatty** [comment]
Fix is in release 0.17.0
